src/server/gc_worker/gc_worker.rs

```diff
@@ -817,6 +818,45 @@ where
                 // Warn and ignore it.
                 warn!("unsafe destroy range: failed sending ClearRegionSizeInRange"; "err" => ?e);
             });
+        } else {
+            let cfs = &[CF_LOCK, CF_DEFAULT, CF_WRITE];
+            let keys = vec![start_key.clone(), end_key.clone()];
+            let regions = get_regions_for_gc(self.store_id, &keys, regions_provider)?;
+            let count = regions.len();
+
+            let mut region_modifies = HashMap::default();
+            for i in 0..count {
```

Comment:
consider: `for (i, region) in regions.into_iter().enumerate()`

Reply:
Good suggestion.